src/pages/index.tsx

```diff
@@ -92,8 +83,27 @@ const Home: NextPage<DefaultSeoInfo> = (props) => {
 
 export async function getServerSideProps(
 	context: GetServerSidePropsContext
-): Promise<GetServerSidePropsResult<DefaultSeoInfo>> {
-	return getDefaultServerSideProps(context);
+): Promise<GetServerSidePropsResult<HomepageProps>> {
+	let title: string | null = null;
+	try {
+		const [contentPage] = await Promise.allSettled([
+			ContentPageService.getBySlug(('/' + context.query.slug) as string),
```

Comment:
For this ContentPageService you need to use the client side version, since the getServerSideProps is executed during prerendering of the pages and at that time the admin core config is not set yet.
```
import { ContentPageService } from '../modules/content-page/services/content-page.service';
```
instead of
```
import { ContentPageService } from '@meemoo/admin-core-ui';
```